controllers/user.js

```diff
@@ -66,22 +69,28 @@ class Users {
     * @param {object} res - The response object.
     * @param {object} next - The response object.
     */
-  static login(req, res, next) {
-    passport.authenticate('local', { session: false }, (
-      err,
-      user
-    ) => {
-      if (err) {
-        return next(err);
-      }
-
-      if (user) {
-        return res.json({ message: 'Login was successful' });
+  static async login(req, res) {
+    const loginUser = await User
+      // eslint-disable-next-line max-len
+      .findOne({ where: { [or]: [{ username: { [iLike]: req.body.username } }, { email: { [iLike]: req.body.email } }] } });
+    if (loginUser) {
+      if (bcrypt.compareSync(req.body.password, loginUser.password)) {
```

Comment:
There is a `validPassword` method available on the Prototype of the `User` model. The function encapsulates the password checking functionality. That's what you should use to check the password here. No need to import `bcrypt`.

TLDR: Check out the `validPassword` method in the `User` model for more context.

Reply:
ok.thanks.